frontend/src/Homepage.css

```diff
@@ -0,0 +1,28 @@
+nav {
+
+    display: flex;
+    justify-content: space-evenly;
+    gap: 12px;
+    position: fixed;
+    bottom: 0;
+    background-color: white;
+    width: 100vw;
+    overflow: scroll;
+    padding-left: 12px;
+    padding-right: 12px;
+    border: black solid 1px;
+
+
+}
+
+.NavLink {
```

Comment:
I'm not sure if the NavLinks behave like links if you set a custom class. If not, consider adding `cursor: pointer`